test/Hooks.t.sol

```diff
@@ -25,11 +25,12 @@ contract HooksTest is Test, Deployers, GasSnapshot {
     using PoolIdLibrary for PoolKey;
     using Hooks for IHooks;
 
-    address payable ALL_HOOKS_ADDRESS = payable(0xfFf0000000000000000000000000000000000000);
+    /// 1111 1111 1100
+    address payable ALL_HOOKS_ADDRESS = payable(0xFfC0000000000000000000000000000000000000);
     MockHooks mockHooks;
 
     // Update this value when you add a new hook flag. And then update all appropriate asserts.
-    uint256 hookPermissionCount = 12;
```

Comment:
Why was it 12? 🤔 was it just out of date before?

Reply:
I think it was out of date. Im guessing when we merged access lock this didn't get updated